cypress/integration/plugins/dashboards-flow-framework/create_workflow_spec.js

```diff
@@ -182,16 +170,8 @@ describe('Creating Workflows Using Various Methods', () => {
       .click();
 
     cy.fixture(FF_FIXTURE_BASE_PATH + 'semantic_search/search_response').then(
```

Comment:
- Same as above. 
- And also remove the cy.fixture line in here. As fixture is no longer needed.

Reply:
same as above.